tests/ControllerTest.cpp

```diff
@@ -0,0 +1,236 @@
+    int initCount = controller.getLeftCount();
+    controller.setLeftMotorPower(power);
+    int currentCount = controller.getLeftCount();
+    EXPECT_LT(initCount, currentCount);
+    controller.resetLeftMotorPower();
+  }
+
+  // 左車輪のモータにマイナスのPower値をセットできるかどうかのテスト
+  TEST(ControllerTest, setLeftMotorMinusPower)
+  {
+    // Measurer measurer;
+    Controller controller;
+    const int power = -50;
+    int initCount = controller.getLeftCount();
+    controller.setLeftMotorPower(power);
+    int currentCount = controller.getLeftCount();
+    EXPECT_GT(initCount, currentCount);
+    controller.resetLeftMotorPower();
+  }
+
+  // モータに設定するPower値の制限が行われているか確認するテスト
+  TEST(ControllerTest, setLeftMotorPowerMin)
+  {
+    Controller controller;
+    const int power = -150;
+    controller.setLeftMotorPower(Controller::MOTOR_POWER_MIN);
+    int minCount = controller.getLeftMotorPower();
+    controller.resetLeftMotorPower();
+    controller.setLeftMotorPower(power);
+    int currentCount = controller.getLeftMotorPower();
+    EXPECT_EQ(minCount, currentCount);
+    controller.resetLeftMotorPower();
+  }
+
+  // stopWheelsMotor()を呼び出せるか確認するテスト
+  TEST(ControllerTest, stopWheelsMotor)
+  {
+    Controller controller;
+    controller.stopWheelsMotor();
+    SUCCEED();
+  }
+
+  // brakeWheelsMotor()を呼び出せるか確認するテスト
+  TEST(ControllerTest, brakeWheelsMotor)
+  {
+    Controller controller;
+    controller.brakeWheelsMotor();
+    SUCCEED();
+  }
+
+  // アームモータにPower値を設定できるかどうか確認するテスト
+  TEST(ControllerTest, setArmMotorPower)
+  {
+    Controller controller;
+    const int power = 50;
+    int initCount = controller.getArmMotorCount();
+    controller.setArmMotorPower(power);
+    int currentCount = controller.getArmMotorCount();
+    EXPECT_LT(initCount, currentCount);
+    controller.resetArmMotorPower();
+  }
+
+  // stopArmMotor()を呼び出せるか確認するテスト
+  TEST(ControllerTest, stopArmMotor)
+  {
+    Controller controller;
+    controller.stopArmMotor();
+    SUCCEED();
+  }
+
+  // holdArmMotor()を呼び出せるか確認するテスト
+  TEST(ControllerTest, holdArmMotor)
+  {
+    Controller controller;
+    controller.holdArmMotor();
+    SUCCEED();
+  }
+
+  // 右タイヤのPower値を取得できるかのテスト
+  TEST(ControllerTest, getRightMotorPwm)
+  {
+    Controller controller;
+    int expected = 0;
+    int actual = controller.getRightMotorPower();
+    EXPECT_EQ(expected, actual);
+    int power = 90;
+    controller.setRightMotorPower(power);
+    expected = 90;
+    actual = controller.getRightMotorPower();
+    EXPECT_EQ(expected, actual);
+    power = 200;
+    controller.setRightMotorPower(power);
+    expected = 100;
+    actual = controller.getRightMotorPower();
+    EXPECT_EQ(expected, actual);
+    power = -200;
+    controller.setRightMotorPower(power);
+    expected = -100;
+    actual = controller.getRightMotorPower();
+    EXPECT_EQ(expected, actual);
+    controller.resetRightMotorPower();
+  }
+
+  // 左タイヤのPower値を取得できるかのテスト
+  TEST(ControllerTest, getleftMotorPower)
```

Comment:
```suggestion
  TEST(ControllerTest, getLeftMotorPower)
```
テストケース名がキャメル担ってない箇所あり。
left -> Left